docs-src/app-dev-context/ide-debug.md

```diff
@@ -0,0 +1,14 @@
+---
+id: ide-debug
+title: Debug with IDE
+description: How to debug your code using your favorite IDE
+sidebar_label: Debug with IDE
+tags:
+  - guide-context
+---
+
+Many IDEs support a debugging feature that enables you to step through your code.
+You can use these tools with your Workflow code, however make sure to set the `TEMPORAL_DEBUG` environment variable to `true` first.
```

Comment:
Python also allows you to set `debug_mode=True` and TypeScript also allows `debugMode: true` as alternatives to environment variables.

Reply:
In either the replayer or worker on each. So for TypeScript it's [debugInfo](https://typescript.temporal.io/api/interfaces/worker.WorkerOptions#debugmode) on `Worker.create` (worker and replayer combined in that language). For Python, it's `debug_mode` on [Worker.__init__](https://python.temporal.io/temporalio.worker.Worker.html#__init__) or [Replayer.__init__](https://python.temporal.io/temporalio.worker.Replayer.html#__init__).

If the env var is set, it will be the same as setting those to true in Python and [soon in TypeScript](https://github.com/temporalio/sdk-typescript/issues/1020).

Reply:
Slight correction, it's `debugMode` for TS.